sdcm/cluster.py

```diff
@@ -2600,6 +2600,8 @@ def check_node_health(self, retries: int = CHECK_NODE_HEALTH_RETRIES) -> None:
             event = next(events, None)
             if event is None:
                 LOGGER.debug("Node `%s' is healthy", self.name)
+                for event in events:
+                    event.dont_publish()
```

Comment:
Don't need this because events generator is exhausted. This two lines is no-op effectively.

Reply:
True